### PR TITLE
Fix scale to zero example

### PR DESCRIPTION
**Pull request by kwiatekus**
**Description**

Changes proposed in this pull request:

- use recommended `emitCloudEvent` way to send CE from a function
- fix test command - content type matching the actual content in the curl command
- fix git reference for the sample functions

### CODE REVIEW

**Review by pPrecel — DISMISSED** [review]

File: scale-to-zero-with-keda/README.md [hunk 1]
```diff
@@ -19,12 +19,16 @@ Keda is used to scale the worker Function. [KEDA Prometheus scaler](https://keda
 
 ## Prerequisites
 
-- Kyma as the target deployment environment.
-- Keda module enabled
+- Kyma as the target kubernetes runtime.
```
Comment (grego952) [inline scale-to-zero-with-keda/README.md:22]:
```suggestion
- Kyma as the target Kubernetes runtime.
```

File: scale-to-zero-with-keda/README.md [hunk 2]
```diff
@@ -19,12 +19,16 @@ Keda is used to scale the worker Function. [KEDA Prometheus scaler](https://keda
...
+- Keda module [installed](https://github.com/kyma-project/keda-manager#install-keda-manager-and-keda-from-the-latest-release).
```
Comment (grego952) [inline scale-to-zero-with-keda/README.md:23]:
```suggestion
- [Keda module installed](https://github.com/kyma-project/keda-manager#install-keda-manager-and-keda-from-the-latest-release)
```

File: scale-to-zero-with-keda/README.md [hunk 3]
```diff
@@ -19,12 +19,16 @@ Keda is used to scale the worker Function. [KEDA Prometheus scaler](https://keda
...
+- Serverless module [installed](https://github.com/kyma-project/serverless-manager#install)
```
Comment (grego952) [inline scale-to-zero-with-keda/README.md:24]:
```suggestion
- [Serverless module installed](https://github.com/kyma-project/serverless-manager#install)
```

File: scale-to-zero-with-keda/README.md [hunk 4]
```diff
@@ -19,12 +19,16 @@ Keda is used to scale the worker Function. [KEDA Prometheus scaler](https://keda
...
+- Custom prometheus stack [installed](https://github.com/kyma-project/examples/blob/main/prometheus/README.md#installation) with istio scraping [enabled](https://github.com/kyma-project/examples/blob/main/prometheus/README.md#installation)
```
Comment (grego952) [inline scale-to-zero-with-keda/README.md:25]:
```suggestion
- [Custom Prometheus stack installed](https://github.com/kyma-project/examples/blob/main/prometheus/README.md#installation) with [Istio scraping enabled](https://github.com/kyma-project/examples/blob/main/prometheus/README.md#installation)
```

File: scale-to-zero-with-keda/README.md [hunk 5]
```diff
@@ -19,12 +19,16 @@ Keda is used to scale the worker Function. [KEDA Prometheus scaler](https://keda
...
+
+>> NOTE: Serverless and prometheus are included in kyma in versions up to 2.19.x
```
Comment (grego952) [inline scale-to-zero-with-keda/README.md:27]:
```suggestion
>> NOTE: Serverless and Prometheus are included in Kyma versions up to 2.19.x
```

File: scale-to-zero-with-keda/README.md [hunk 6]
```diff
@@ -19,12 +19,16 @@ Keda is used to scale the worker Function. [KEDA Prometheus scaler](https://keda
...
 
 ## Installation
 
-Make sure istio sidecar injection is enabled in the target Namesapce:
+Make sure istio sidecar injection is enabled in the target Namespace:
```
Comment (grego952) [inline scale-to-zero-with-keda/README.md:31]:
```suggestion
Make sure Istio sidecar injection is enabled in the target Namespace:
```

**Review by grego952 — APPROVED** [review]

**Review by pPrecel — APPROVED** [review]

**Comment by qooksgit** [comment]
Okay, this PR has been merged. But still, this does not fix my issue. It is not still working on my side. I have posted its status in the slack thread.   
@kwiatekus  Can you visit the thread?
https://sap-data-intelligence.slack.com/archives/C0140PCSJ5Q/p1698674450246869?thread_ts=1698211631.843389&cid=C0140PCSJ5Q

**Pull request merged**